x-pack/plugins/security_solution/public/management/pages/policy/view/artifacts/empty/policy_artifacts_empty_unexisting.tsx

```diff
@@ -43,28 +39,27 @@ export const PolicyArtifactsEmptyUnexisting = memo<CommonProps>(
       }
     );
     return (
-      <EuiPageTemplate template="centeredContent">
-        <EuiEmptyPrompt
-          iconType="plusInCircle"
-          data-test-subj="policy-artifacts-empty-unexisting"
-          title={<h2>{labels.emptyUnexistingTitle}</h2>}
-          body={labels.emptyUnexistingMessage}
-          actions={
-            canWriteArtifact ? (
-              // eslint-disable-next-line @elastic/eui/href-or-on-click
-              <EuiButton
-                color="primary"
-                fill
-                onClick={onClickHandler}
-                href={toRouteUrl}
-                data-test-subj="unexisting-manage-artifacts-button"
-              >
-                {labels.emptyUnexistingPrimaryActionButtonTitle}
-              </EuiButton>
-            ) : null
-          }
-        />
-      </EuiPageTemplate>
+      <EuiEmptyPrompt
```

Comment:
Same question here. Should this be nested within `EuiPageTemplate`?

Reply:
see this comment: https://github.com/elastic/kibana/pull/162531#discussion_r1307861905